spec/requests/requests_requests_spec.rb

```diff
@@ -43,15 +43,17 @@
       end
 
       context "when there is a filter applied" do
-        it "shows print unfulfilled picklists button with correct quantity when filtered" do
+        it "shows only filtered requests, print unfulfilled picklists button with correct quantity" do
           Request.delete_all
 
-          create(:request, :pending)
-          create(:request, :started)
+          started_request = create(:request, :started, comments: "Started request - should appear")
+          pending_request = create(:request, :pending, comments: "Pending request - should not appear")
 
           get requests_path({ filters: { by_status: :started} })
 
           expect(response.body).to include('Print Unfulfilled Picklists (1)')
+          expect(response.body).to include(started_request.comments)
```

Comment:
Please use the actual string that's expected - whenever possible, expected results should not be calculated.